src/app/shared/image/image-loader.component.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { cit, create, cms } from '../../../testing';
+import { ImageLoaderComponent } from './image-loader.component';
+
+describe('ImageLoaderComponent', () => {
+
+  create(ImageLoaderComponent);
+
+  const waitFor = (comp, event, callback) => {
+    let originalFn = comp[event];
+    spyOn(comp, event).and.callFake(function() {
+      originalFn.apply(comp, arguments);
+      callback();
+    });
+  };
+
+  const getBackground = (el) => {
+    let style = el.nativeElement.getAttribute('style') || '';
+    let match = style.match(/background-image: url\((.+)\)/);
+    return match ? match[1] : '';
+  };
+
+  const mockDoc = (linkHref) => {
+    let doc = cms.mock('prx:anything', {});
+    if (linkHref) {
+      doc.mock('prx:image', {_links: {enclosure: {href: linkHref}}});
+    }
+    return doc;
+  };
+
+  describe('from src', () => {
+
+    cit('renders the image src', (fix, el, comp, done) => {
+      comp.src = 'http://placehold.it/10x10';
+      waitFor(comp, 'onLoad', () => {
+        expect(el).toQueryAttr('img', 'src', 'http://placehold.it/10x10');
+        expect(getBackground(el)).toEqual('http://placehold.it/10x10');
```

Comment:
This test isn't passing for me. I'm getting `"`'s around the background-image url. Seems odd because we're not putting those quotes in the component's setBackgroundImage(src)
`1) renders the image src
     ImageLoaderComponent from src
     Expected '"http://placehold.it/10x10"' to equal 'http://placehold.it/10x10'.
    at webpack:///Users/sandikbarr/code/prx/publish.prx.org/src/app/shared/image/image-loader.component.spec.ts:36:34 <- src/test.ts:34166:43
    at ImageLoaderComponent.<anonymous> (webpack:///Users/sandikbarr/code/prx/publish.prx.org/src/app/shared/image/image-loader.component.spec.ts:12:6 <- src/test.ts:34146:13)`